apps/client/src/models/tags/text.ts

```diff
@@ -0,0 +1,13 @@
+export type TextTypeTag =
+  | 'Display'
+  | 'Title'
+  | 'Label'
+  | 'Paragraph'
+  | 'Heading';
+export type TextSizeTag =
+  | '2XSmall'
+  | 'XSmall'
+  | 'Small'
+  | 'Medium'
+  | 'Large'
+  | 'XLarge';
```

Comment:
IMHO these should be in lowercase. Can you discuss with @pasinducw

Reply:
Also if it need to be lowercase. do the change in a separate future PR since there are work based on this

Reply:
Can you use enums instead of using plain text values?
Eg:
```
enum TextVariant {
   Display = 'display',
  Title = 'title',
  ...
}

enum TextSize {
  Small = 'small',
  ExtraSmall = 'x-small',
  DoubleExtraSmall = '2x-small',
  ...
}
```

Reply:
refactored text component https://github.com/the-ai-team/razor/pull/51/commits/9953c3957419625c23d0ca0cccf993ac3ea6eea9